Library/Homebrew/cask/lib/hbc/dsl/appcast.rb

```diff
@@ -9,6 +11,24 @@ def initialize(uri, parameters = {})
         @checkpoint     = @parameters[:checkpoint]
       end
 
+      def calculate_checkpoint
+        result = SystemCommand.run("/usr/bin/curl", args: ["--compressed", "--location", "--user-agent", URL::FAKE_USER_AGENT, @uri], print_stderr: false)
+
+        checkpoint = if result.success?
+          processed_appcast_text = result.stdout.gsub(%r{<pubDate>[^<]*</pubDate>}, "")
+
+          # This step is necessary to replicate running `sed` from the command line
```

Comment:
I just thought I'd leave this in for now as to not mess up existing values. But I guess this doesn't make any difference since it isn't officially used for anything yet.